query/common.d.ts

```diff
@@ -269,6 +277,39 @@ export interface Bindings extends Iterable<[RDF.Variable, RDF.Term]> {
   ) => Bindings;
 }
 
+export interface Paths extends Iterable<Bindings> {
```

Comment:
```suggestion
export interface Path extends Iterable<Bindings> {
```

I really think this should be singular.